crispy_forms/templatetags/crispy_forms_field.py

```diff
@@ -132,6 +132,7 @@ def render(self, context):
             if (
                 template_pack == 'bootstrap4'
                 and not is_multivalue(field)
+                and not is_file(field)
```

Comment:
Assuming we take #920, what do we do about this?

Reply:
Indeed we do not need it anymore, as it will be handled in https://github.com/django-crispy-forms/django-crispy-forms/pull/920/files#diff-0f9e2d08d3884dbef59475bf2552fa0eR38

Reply:
OK, let me have a play with it all...